lib/miq_automation_engine/engine/miq_ae_object.rb

```diff
@@ -116,7 +116,7 @@ def initialize(workspace, ns, klass, instance, object_name = nil)
 
       Benchmark.realtime_block(:inherit_time) do
         # Who do we inherit from
-        @inherits = @aec.inherits.blank? ? BASE_OBJECT : @aec.inherits
+        @inherits = BASE_OBJECT
```

Comment:
@chessbyte 
The class inheritance was disabled once we added multiple domains support, because if the same class exists in multiple domains with different schemas which schema would get used. The CFME UI other than letting a user select a schema from a drop down list of all classes, didn't allow the user to create an instance with fields defined in the current and parent class, it would only allow you to create an instance with the current class schema. So it was partially implemented in the UI. The Automate Engine had the ability to do the inheritance from the different class if the "inherits" value is defined. We have had some old customers who have the old database that still has the "inherits" value defined and whenever they import the old database, the Automate Engine fails trying to locate the classes across the domain. This code disables the inheritance so that we won't go looking for the class across domains if we find legacy data.